content/en/docs/setup/independent/high-availability.md

```diff
@@ -10,513 +10,217 @@ content_template: templates/task
 
-4. Restart keepalived. While no Kubernetes services are up yet it will log health check fails on all master nodes. This will stop as soon as the first master node has been bootstrapped.
-{{% /tab %}}
-{{< /tabs >}}
+# External Etcd -- The instructions
 
+This guide assumes calico will be used as the pod network and will therefore use
+some default values, but they will be called out explicitly so replacing calico
+with another pod network is possible.
 
-## Acquire etcd certs
+## Setting up an HA etcd cluster using kubeadm
 
-Only follow this step if your etcd is hosted on dedicated nodes (**Option 1**). If you are hosting etcd on the masters (**Option 2**), you can skip this step since you've already generated the etcd certificates on the masters.
+Please see [setting up an HA etcd cluster using kubeadm](/docs/tasks/administer-cluster/setup-etcd-with-kubeadm).
 
-1. Generate SSH keys for each of the master nodes by following the steps in the [create ssh access](#create-ssh-access) section. After doing this, each master will have an SSH key in `~/.ssh/id_rsa.pub` and an entry in `etcd0`'s `~/.ssh/authorized_keys` file.
+After this setup you should have these files on the first bootstrapped member.
+You will need to put these files on all of your control-plane nodes.
 
-1. Run the following:
-     ```bash
-     mkdir -p /etc/kubernetes/pki/etcd
-     scp root@<etcd0-ip-address>:/etc/kubernetes/pki/etcd/ca.pem /etc/kubernetes/pki/etcd
-     scp root@<etcd0-ip-address>:/etc/kubernetes/pki/etcd/client.pem /etc/kubernetes/pki/etcd
-     scp root@<etcd0-ip-address>:/etc/kubernetes/pki/etcd/client-key.pem /etc/kubernetes/pki/etcd
-     ```
+1. `/etc/kubernetes/pki/etcd/ca.crt`
+1. `/etc/kubernetes/pki/apiserver-etcd-client.crt`
+1. `/etc/kubernetes/pki/apiserver-etcd-client.key`
 
-## Run `kubeadm init` on `master0` {#kubeadm-init-master0}
+Here is one way to do it, but there are many reasons why this might not work for
+you. For instance, you may be running any other linux distro.
 
-1. In order for kubeadm to run, you first need to write a configuration file:
-     ```bash
-     cat >config.yaml <<EOF
-     apiVersion: kubeadm.k8s.io/v1alpha1
-     kind: MasterConfiguration
-     api:
-      advertiseAddress: <private-ip>
-     etcd:
-      endpoints:
-      - https://<etcd0-ip-address>:2379
-      - https://<etcd1-ip-address>:2379
-      - https://<etcd2-ip-address>:2379
-      caFile: /etc/kubernetes/pki/etcd/ca.pem
-      certFile: /etc/kubernetes/pki/etcd/client.pem
-      keyFile: /etc/kubernetes/pki/etcd/client-key.pem
-     networking:
-      podSubnet: <podCIDR>
-     apiServerCertSANs:
-     - <load-balancer-ip>
-     apiServerExtraArgs:
-      apiserver-count: "3"
-     EOF
-     ```
+```
+scp /etc/kubernetes/pki/etcd/ca.crt ubuntu@ETCD_HOST_1:/home/ubuntu
+scp /etc/kubernetes/pki/apiserver-etcd-client.crt ubuntu@ETCD_HOST_1:/home/ubuntu
+scp /etc/kubernetes/pki/apiserver-etcd-client.key ubuntu@ETCD_HOST_2:/home/ubuntu
+```
 
```

Comment:
Add note here:

"This is an example only. Your environment or Linux distribution might require different commands."